### ISSUE TITLE
mod_md still intercepts requests for challenges after disabling `MDomain`

### ISSUE THREAD

**Issue by djc** [icing/mod_md#93]
After testing with one of my domains, I commented out `MDomain` for that domain. However, if I request `https://example.com/.well-known/acme-challenge/`, I still get the value from `/usr/lib64/apache2/md/challenges/example.com` rather than the one from `/var/www/example.com/http/.well-known/acme-challenge`. This means my previous method of getting new certificates is failing.

**Comment by icing** [comment]
Does this work again when you remove the ```md/challenges/<domain>``` content?

**Comment by icing** [comment]
And by reporting here you are using the latest version from github, I assume?

**Comment by icing** [comment]
Please, if you update, give the lastest version v1.1.13 a try.

**Comment by djc** [comment]
Ah, sorry, no, I was still using the version delivered with httpd 2.4.33. I now installed 1.1.13 from source (I'm assuming/hoping the whole patching of mod_ssl dance is no longer necessary) and it's interestingly a little better but not quite there: the `/.well-known/acme-challenge/` path presents as empty/404, but when I try request any file within it (for example, another challenge that I placed in the correct directory), I still get the challenge value from `md/domains`. Removing those files did help and I was able to reverify the domain using my other ACME client. Perhaps mod_md should remove those files on configuration changes if `MDomain` is not enabled?
